runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -338,15 +338,18 @@ public void onGameTick(GameTick event)
 
 		executorService.submit(() ->
 		{
-			try
+			if (!geText.getText().contains(OSB_GE_TEXT)) 
```

Comment:
you should flip this if so the diff will be small (e.g if (geText.contains)... return